docs/working-with-media-collections/defining-media-collections.md

```diff
@@ -106,10 +106,12 @@ public function registerMediaCollections()
 {
     $this
         ->addMediaCollection('avatar')
-        ->singleFile();
+        ->onlyKeepLatests(1);
```

Comment:
SHouldn't it stay `->singleFile()` and add an example with `->onlyKeepLatests(5)` to showcase the difference?

Reply:
Check. Added a seperate example in the docs as "Limited file collections" under "Single file collections".

Reply:
https://github.com/spatie/laravel-medialibrary/pull/1533/commits/026b43f4e479b0cd574633d081cbe7d6189b9cbc